### ISSUE TITLE
when enabling grid view every video showing in a single column 

### ISSUE THREAD

**Issue by devv2006** [Fugiman/google-meet-grid-view#243]
_Before opening an issue, please check the existing open issues to see if somebody has already reported it! If something is horribly wrong, like all videos showing up in a single column, chances are somebody already reported it. Please reply in that issue instead._

**Describe what is broken**
A clear and concise description of what isn't working.

**Expected behavior**

**Screenshots**

**Software used:**
 - OS [e.g. Windows 10]:
 - Browser [e.g. Chrome, Firefox]:
 - Version [e.g. 22]:
 - Extension [e.g. Official Chrome Extension, Tampermonkey]:
 - Grid View Version [e.g. 1.19, 1.23]:

**Additional context**

**Comment by devv2006** [comment]
plzz help

**Comment by devv2006** [comment]
its a huge issue!!!!!!!

**Comment by devv2006** [comment]
i have been asking for help since so long

**Comment by devv2006** [comment]
no need, i removed the extension,


highly annoyed and disappointed, by such a bad service.


Did not expected it from github

**Comment by devv2006** [comment]
looking for another good extension, other than this,

worst experience

**Comment by RiZKiT** [comment]
Seems to be a duplicate of https://github.com/Fugiman/google-meet-grid-view/issues/242.

**Comment by ThomasQvist** [comment]
> looking for another good extension, other than this,
> 
> worst experience

Please remember that this is a completely free piece of software - If you are unsatisfied - please write your own code

**Comment by gjh0239** [comment]
> no need, i removed the extension,
> 
> highly annoyed and disappointed, by such a bad service.
> 
> Did not expected it from github

You are not entitled to a response within an hour. Please consider making a pull request if you feel like you are capable of fixing it. If not, just wait like everyone else is doing. Thank you.

**Comment by adamduff** [comment]
> Seems to be a duplicate of #242.

Yes it is the same problem, but it had been resolved - now seems to be back again

**Comment by tankrizzo** [comment]
> no need, i removed the extension,
> 
> highly annoyed and disappointed, by such a bad service.
> 
> Did not expected it from github

It is not a "service". You do not pay anything for this. It is a project by a single developer giving up his own time to make it. Have some patience and perhaps some manners.

**Comment by maxburboav** [comment]
Efectivamente el Grid, no está funcionando desde hoy 1° de julio. Agradecería que se pudiera solucionar.
Consulta hay alguna opción de configuración previa en Chrome como para evitar las caídas?

Saludos

**Comment by vretamal** [comment]
> Efectivamente el Grid, no está funcionando desde hoy 1° de julio. Agradecería que se pudiera solucionar.
> Consulta hay alguna opción de configuración previa en Chrome como para evitar las caídas?
> 
> Saludos

No, Google meet se actualiza desde el lado del servidor, por lo que no nos daremos cuenta de las actualizaciones hasta que algo rompen en esta extensión

_Translated:
No, Google meet has server-side updates, so we don't know when it's updated until it break something in the extension._

**Comment by soporte-ti** [comment]
The same problem

OS: KDE Neon
Browser: Google Chrome 83.0.4103.116
Meet Grid View: 1.37.0
![image](https://user-images.githubusercontent.com/66955026/86249875-795e1c80-bb75-11ea-87fa-c18c039cf457.png)

**Comment by ncbaratta** [comment]
Same for me

OS: Red Hat Enterprise Linux 7.7
Browser: Version 83.0.4103.116 (Official Build) (64-bit)
Meet Grid View: 1.37.0

**Comment by AlexP11223** [comment]
> looking for another good extension, other than this,
> 
> worst experience

Do you realize that

- It was broken because of changes made by Google. All other unofficial extensions have the same risk of breaking at any time.
- It is a free product, no warranty for quick support.

? :slightly_smiling_face:

**Comment by mikeredgrave** [comment]
> no need, i removed the extension,
> 
> highly annoyed and disappointed, by such a bad service.
> 
> Did not expected it from github

A) It's free. Stop whining.
B) Github host code. they do not produce code. 
C) Google obviously made an API change.

**Comment by kajalsanklecha** [comment]
Is this issue OS specific or maybe browser specific, please do check.

The same issue on my system too.

Ubuntu 20.04 LTS
Google Chrome
